bin/release-docker.sh

```diff
@@ -33,11 +33,9 @@ fi
 set -ex
 
 for image in $images; do
+    bazel $BAZEL_ARGS run //docker:$image
     for tag in $tags; do
-        bazel $BAZEL_ARGS run //docker:$image $hub/$image:$tag
+        docker tag istio/docker:init $hub/$image:$tag
```

Comment:
Oops, this should be $image. @sebastienvas may end up taking over this PR anyway as there might need to be some additional changes in istio-testing to make this work properly. Keeping this PR for now as a reference.